### DISCUSSION TITLE
Which docker image?

### DISCUSSION THREAD

**Discussion by miran248** [hasura/graphql-engine#10145] [Question] [answered]
Hey,

Which image should be used these days if you have a pro license?
[hasuraci/graphql-engine](https://hub.docker.com/r/hasuraci/graphql-engine),
[hasuraci/graphql-engine-pro](https://hub.docker.com/r/hasuraci/graphql-engine-pro) or
[hasura/graphql-engine](https://hub.docker.com/r/hasura/graphql-engine)?

Also, what's the difference between the pro and enterprise?
Can i just switch from `hasuraci/graphql-engine-pro` to `hasura/graphql-engine` and change the `HASURA_GRAPHQL_PRO_KEY` to `HASURA_GRAPHQL_EE_LICENSE_KEY`?

Thanks

**Comment by cmartin616** [comment, marked as answer]
@miran248 - I am assuming you are on a later version than 2.11 (previous LTS release) so you should be using [hasura/graphql-engine](https://hub.docker.com/r/hasura/graphql-engine) as your image.  [hasuraci/graphql-engine-pro](https://hub.docker.com/r/hasuraci/graphql-engine-pro) stopped publishing images after 2.11 except for 2.11 patches.

The appropriate license and env var to use is going to depend on your Hasura setup.  If you are using the legacy Hasura Control Plane (Lux) then you should continue to use `HASURA_GRAPHQL_PRO_KEY` as Lux handled many of the Enterprise features.  If you are not using Lux then you should use `HASURA_GRAPHQL_EE_LICENSE_KEY` to unlock the Enterprise features within the Hasura GraphQL Engine (data plane).  Keep in mind the Pro Key is generated by Lux whereas the EE license key is generated by Hasura Support.

Please let me know if you have further questions.
